_sass/base/_typography.scss

```diff
@@ -212,7 +223,38 @@ hr {
 }
 
 center p a {
+	&:hover{
+		border-bottom: 2px solid _palette(orange);
```

Comment:
This should be removed as we're not using underlines on any other links at this time.

It will add the underline i.e., to the text on the TL App User Guide where the image is centered and has text below it.